soc/xtensa/intel_s1000/memory.h

```diff
@@ -6,30 +6,40 @@
 #ifndef __INC_MEMORY_H
 #define __INC_MEMORY_H
 
+#include <autoconf.h>
+
 /* L2 HP SRAM */
 #define L2_VECTOR_SIZE				0x1000
 
+#ifdef CONFIG_BOOTLOADER_MCUBOOT
```

Comment:
Is this really something that should be specified by the SoC?  Seems like more a board choice.

Reply:
CONFIG_BOOTLOADER_MCUBOOT is a board choice. But, the memory range is defined only in this file which is in SoC directory.